### DISCUSSION TITLE
How to use mobile number to sign in user

### DISCUSSION THREAD

**Discussion by SC0d3r** [nextauthjs/next-auth#1360] [Help]
I want to use mobile number to login a user but where should I start the process should I make a callback ? and when user doesnt exists how should I create user in db ? does next-auth create the user or should I ?, I noticed when user gets created in db for first time with Email it has some fields like createdAt ,... when the user signs in for the first time should I create those or next will create them for me and in general how should I handle the entire process? Thanks in advance

**Comment by iamthesiz** [comment]
I'm currently working on this too. I'm still working on **how to get the user's created** when they aren't already there, but these were helpful starts.
https://everify.dev/blog/two-factor-authentication-for-nextjs
https://github.com/nextauthjs/next-auth/issues/709#issuecomment-784077142
